modules/virt-customizing-storage-profile.adoc

```diff
@@ -67,3 +67,37 @@ status:
 <2> The `volumeMode` that you select.
 +
 After you save your changes, the selected values appear in the storage profile `status` element.
+
+== Setting a default cloning strategy using a storage profile
+
+You can use storage profiles to set a default cloning method for a storage class, creatng a _cloning strategy_. Setting cloning strategies  can be helpful, for example, if your storage vendor only supports certain cloning methods. It also allows you to select a method that limits resource usage or maximizes performance.
+
+Cloning strategies can be specified by setting the `cloneStrategy` attribute in a storage class to one of these values:
+
+* `snapshot` - The default cloning method, using smart cloning. Smart cloning quickly and efficiently creates a clone by cloning the data volume PVC to another PVC.
+* `copy` - Host-assisted cloning, used if your storage vendor does not support smart cloning. Host-assisted cloning is the least efficient method of cloning.
+* `csi-clone` - CSI volume cloning. CSI volume clones copy an existing volume with the efficiency and performance advantages of smart cloning. Unlike `snapshot` or `copy`, which are used by default if no storage profile is defined, CSI volume cloning is only used when you specify it in the `StorageProfile` object for the provisioner's storage class.
```

Comment:
Uses the CSI clone API to efficiently clone an existing volume without using an interim volume snapshot.

Reply:
Changed to: 

* `csi-clone` - This method uses the CSI clone API to efficiently clone an existing volume without using an interim volume snapshot. Unlike `snapshot` or `copy`, which are used by default if no storage profile is defined, CSI volume cloning is only used when you specify it in the `StorageProfile` object for the provisioner's storage class.

Let me know if more changes are needed.